openmetadata-ui/src/main/resources/ui/playwright/utils/sidebar.ts

```diff
@@ -17,6 +17,11 @@ import {
 } from '../constant/settings';
 import { SidebarItem, SIDEBAR_LIST_ITEMS } from '../constant/sidebar';
 
+export const clickOnLogo = async (page: Page) => {
+  await page.click('#openmetadata_logo > [data-testid="image"]');
+  await page.mouse.move(1280, 0); // Move mouse to top right corner
```

Comment:
Why we need to move mouse here?

Reply:
since clicking on the icon opens the sidebar as well and if we want to select something on the left portion it will be hidden by the sidebar.